docs/release-notes/8.3.asciidoc

```diff
@@ -180,6 +180,8 @@ On the next scheduled rule execution, the *Last Response* value for the rule wil
 * Provides an option to select a severity level for a case ({pull}131626[#131626]).
 * Adds the experimental *Alerts* tab to cases, which allows users to inspect attached alerts ({pull}131883[#131883]).
 * Adds the *Average time to close* metric to the Cases page ({pull}131909[#131909]).
+* Adds new fields to prebuilt detection rules' schemas: `related_integrations`, `required_fields`, and `setup` ({pull}132409[#132409]).
+* Adds *Related integrations*, *Required fields*, and *Setup guide* sections to the rule details page, to help users identify, install, and configure a rule's prerequisites. Also adds related integrations badge to the rules table ({pull}131475[#131475]). Content for these new sections is delivered in a prebuilt rules update, independent of {stack} release versioning.
```

Comment:
Are we going to update the 8.3 release notes? Am I understanding correctly that we'll do it only if rules with those fields will be released in one of the `8.3.x` Out-Of-Band updates for the `Prebuilt Detection Rules` package?

Reply:
@banderror Yes, this section will update the 8.3 release notes (the file name is version-specific). 

For 8.4 release notes, we have a [separate PR](https://github.com/elastic/security-docs/pull/2314) that's currently in progress, though it doesn't yet mention the new fields because they're not yet available in the UI. Once I can confirm that, I'll add the same wording from above to those 8.4 notes.

Basically before we merge anything related to the new fields, I'm waiting to confirm that the latest prebuilt rules package has been published and the new fields are available in both 8.3.x and 8.4.0.